### ISSUE TITLE
Laravel 10 compatibility

### ISSUE THREAD

**Issue by MadalinaBrinzea** [vansteen/laravel-sendinblue#10]
Hi,

I upgraded Laravel project to 10, and when I run `composer require vansteen/laravel-sendinblue`,
I receive this error:

`vansteen/laravel-sendinblue[v0.1, ..., v0.4] require illuminate/support ~5 -> found illuminate/support[v5.0.0, ..., v5.8.36] but the package is fixed to v10.13.5 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command`.

Is this package compatible with Laravel 10?

Thank you

**Comment by amanshr72** [comment]
No, The error you're encountering indicates that the package vansteen/laravel-sendinblue has a dependency on illuminate/support version ~5, but your Laravel project is currently using version 10. Laravel 10 introduces significant changes, and packages built for Laravel 5 may not be compatible with Laravel 10 out of the box.

You can Check for an Updated Version or Search for Alternatives
